src/memray/_metadata.py

```diff
@@ -11,6 +11,7 @@ class Metadata:
     peak_memory: int
     command_line: str
     pid: int
+    main_thread_tid: int
```

Comment:
This feels a bit nitpicky, but - I think we should instead call this field
```suggestion
    main_thread_id: int
```

Now that it's going into the metadata it becomes a part of our public interface, and our JSON reports. I worry that using "tid" will mislead people, since it's not a tid in the sense of `gettid()`, or even in the sense of `pthread_self()` - it's a monotonically increasing integer which we've assigned.

Think there's anything to that concern, @pablogsal?

Reply:
I am happy to make this change, will do it tomorrow!